test/server/cards/01_SOR/leaders/SabineWrenGalvanizedRevolutionary.spec.js

```diff
@@ -0,0 +1,44 @@
+describe('Sabine Wren, Galvanized Revolutionary', function() {
+    integration(function() {
+        describe('Sabine\'s undeployed ability', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        leader: 'sabine-wren#galvanized-revolutionary'
+                    }
+                });
+            });
+
+            it('should deal 1 damage to both bases', function () {
+                this.player1.clickCard(this.sabineWren);
+                this.player1.clickPrompt('Deal 1 damage to each base');
+                expect(this.p1Base.damage).toBe(1);
+                expect(this.p2Base.damage).toBe(1);
+                expect(this.sabineWren.exhausted).toBe(true);
+            });
+        });
+
+        describe('Sabine\'s deployed ability', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        leader: { card: 'sabine-wren#galvanized-revolutionary', deployed: true }
+                    },
+                    player2: {
+                        groundArena: ['rebel-pathfinder']
+                    }
+                });
+            });
+
+            it('should deal 1 damage to the opponent\'s base on attack', function () {
```

Comment:
These are great! Only additional test I can think of would be this for a timing test: enemy base at 29 damage, Sabine attacks something, then confirm that when the end game prompt happens she has done 0 damage to the defender (i.e., game ended before her attack resolved).

If we wanted to be extra about it, could also add an additional "on attack" ability with like vambrace grappleshot or something to confirm that game end happens when the player chooses to resolve the Sabine ability. But I'd be fine without that as well, it's not strictly covering anything new.